src/classes/StudentQ.cls

```diff
@@ -0,0 +1,66 @@
+public class StudentQ {
+    public StudentQ() {
+        Student__c student = new Student__c();
+        
+        //Registrar
+        
+        //Personal Information
+        Date myDate = System.today();
+        student.Date_Of_Birth__c = myDate;
+        student.Name = 'Abhay Kevat';
+        student.Roll_Number__c = 19026;
+        student.Father_s_Name__c = 'Mohan Kevat';
+        student.Father_Mobile_Number__c = '9184873834';
+        student.Mobile_Number__c = '9638491657';
+        student.Email__c = 'abhay.kevat@gmail.com';
+        student.Home_Phone__c = '9184873834';
+        student.Overall_Grade__c = 'A';
+        student.Address__c = 'Vrindavan';
+        student.City__c = 'Mathura';
+        student.Pin_Code__c = 281003;
+        student.State__c = 'Uttar Pradesh';
+        student.RecordTypeId = '0125i000000LVYbAAO';
+            
+        //Class Info    
+        student.Section__c = 'B';
+        student.Class__c = 'B.Tech';
+        student.Stream__c = 'Science';
+            
+        //Fees Details
+        student.Current_Year__c = '1st Year';
+        student.Is_Donation__c = False;
+        student.Donation_Amount__c = 0;
+        student.Fees_Submitted__c = True;
+        student.Pending_Fees_Amount__c = 50;
+        student.Exam_Fees_Submitted__c = True;
+        student.Pending_Exam_Fees__c = 0;
+        //student.Pending_Overall_Amount__c = ; //Not Writable
```

Comment:
Make it Writable

Reply:
Sir i use the Formula Data Type for Pending Overall Amount = Pending Fees Amount + Pending Exam Fees